stdlib/websub/src/main/ballerina/src/websub/commons.bal

```diff
@@ -540,6 +540,17 @@ public type Hub object {
     public string publishUrl;
     private http:Listener hubHttpListener;
 
+    # The initialization method for the Hub. Users of the `ballerina/websub` module must use the
+    # function `websub:startHub()` to initialize a `websub:Hub` object.
```

Comment:
```suggestion
    # `websub:startHub()` function to initialize a `websub:Hub` object.
```

Reply:
This is a BFM convention.

Quoting from the spec,

"BFM also provides conventions for referring to Ballerina-defined names from within documentation strings in a source file. An identifier in backticks `X`, when preceded by one of the following words:

- type
- service
- variable
- var
- annotation
- module
- function
- parameter

is assumed to be a reference to a Ballerina-defined name of the type indicated by the word. In the case of parameter, the name must be unqualified and be the name of a parameter of the function to which the documentation string is attached. For other cases, if the name is unqualified it must refer to a public name of the appropriate type in the source file's module; if it is a qualified name M:X, then the source file must have imported M, and X must refer to a public name of an appropriate type in M. BFM also recognizes `f()` as an alternative to function `f`. In both cases, f can have any of the following forms (where `m` is a module import, `x` is a function name, `T` is an object type name, and `y` is a method name):

    x()
    m:x()
    T.y()
    m:T.y()
"

Me using `websub:` was wrong but we can't move "function" to after the function name.

Reply:
+1